tests/test_wave_01.py

```diff
@@ -118,13 +118,9 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
-    
-    raise Exception("Test needs to be completed.")
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
+    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
```

Comment:
Good assertion 👍. You could add a couple more asserts to strengthen this, if you wanted:
```python
    assert updated_data["watched"][0]["genre"] == GENRE_1
    assert updated_data["watched"][0]["rating"] == RATING_1
```